src/SFA.DAS.EmployerIncentives/Infrastructure/Api/PassThroughApiClient.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Infrastructure.Api
+{
+    public class PassThroughApiClient : IPassThroughApiClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ILogger<PassThroughApiClient> _logger;
+
+        public PassThroughApiClient(HttpClient httpClient, ILogger<PassThroughApiClient> logger)
+        {
+            _httpClient = httpClient;
+            _logger = logger;
+        }
+
+        public async Task<InnerApiResponse> Get(Uri uri, object queryData = null, CancellationToken cancellationToken = default)
+        {
+            if (queryData != null)
+            {
+                uri = new Uri(AddQueryString(uri.ToString(), queryData), UriKind.RelativeOrAbsolute);
+            }
+
+            var response = await _httpClient.GetAsync(uri, cancellationToken).ConfigureAwait(false);
```

Comment:
You don't need ConfigureAwait in .net core 3.1 since there is no Synchronization context.

Reply:
Agreed. but in the long term I believe this probably will become a nuget standard package and will therefore need it for framework code